cmd/dns.go

```diff
@@ -85,8 +89,90 @@ func (a *OsintScan) InitDNSCommand() {
 
 	subenumCmd.Flags().String("domain", "", "Domain to get subdomains for")
 
+	takeoverCmd := &cobra.Command{
+		Use:   "takeover",
+		Short: "Detect domain takeovers given targets",
+		Long:  `Detect domain takeovers given targets`,
+		Run: func(cmd *cobra.Command, args []string) {
+			targets, err := cmd.Flags().GetStringSlice("targets")
+			if err != nil {
+				a.OutputSignal.AddError(err)
+				return
+			}
+			filePaths, err := cmd.Flags().GetStringSlice("files")
+			if err != nil {
+				a.OutputSignal.AddError(err)
+				return
+			}
+			fileTargets, err := getTargetsFromFiles(filePaths)
+			if err != nil {
+				a.OutputSignal.AddError(err)
+				return
+			}
+
+			allTargets := append(targets, fileTargets...)
+
+			if len(allTargets) == 0 {
+				a.OutputSignal.AddError(errors.New("no targets specified"))
+				return
+			}
+
+			fingerprintsPath, err := cmd.Flags().GetString("fingerprints")
+			if err != nil {
+				a.OutputSignal.AddError(err)
+				return
+			}
+
+			setHTTP, err := cmd.Flags().GetBool("https")
+			if err != nil {
+				a.OutputSignal.AddError(err)
+				return
+			}
+
+			timeout, err := cmd.Flags().GetInt("timeout")
+			if err != nil {
+				a.OutputSignal.AddError(err)
+				return
+			}
+
+			report, err := dns.DetectDomainTakeover(allTargets, fingerprintsPath, setHTTP, timeout)
+			if err != nil {
+				a.OutputSignal.AddError(err)
+			}
+			a.OutputSignal.Content = report
+		},
+	}
+
+	takeoverCmd.Flags().StringSlice("targets", []string{}, "URL targets to analyze")
+	takeoverCmd.Flags().String("fingerprints", "configs/fingerprints.json", "Abs path to fingerprints file")
```

Comment:
My one last nit on this. Does it actually require an absolute path? Because `configs/fingerprints.json` is not one. Also, can we spell out "Absolute" if that is in fact required? Just makes the docs more professional

Reply:
I use a function `filepath.Abs(string)` that resolves the path to an absolute path by appending the path to the current directory if an absolute path isnt given. This is why configs/fingerprints.json works. Do you think this will confuse the user? Should I remove this and just do a normal os.Readfile on the sent in path

ACK on the "spell out Absolute"